package/MDAnalysis/analysis/lineardensity.py

```diff
@@ -50,24 +50,58 @@ class LinearDensity(AnalysisBase):
           histograms. Defines the resolution of the resulting density
           profile (smaller --> higher resolution)
     verbose : bool (optional)
-          Show detailed progress of the calculation if set to ``True``; the
-          default is ``False``.
+          Show detailed progress of the calculation if set to ``True``
 
     Attributes
     ----------
-    results : dict
-          Keys 'x', 'y', and 'z' for the three directions. Under these
-          keys, find 'pos', 'pos_std' (mass-weighted density and
-          standard deviation), 'char', 'char_std' (charge density and
-          its standard deviation), 'slice_volume' (volume of bin).
+    results.x.dim : int
```

Comment:
Pedantically I'd change all the "in [xyz] direction"s below to "along [xyz] axis"s but up to you :-)

Reply:
Good point.